src/leiningen/new/re_frame/shadow-cljs.edn

```diff
@@ -5,25 +5,16 @@
  :builds {:app {:target          :browser
                 :output-dir      "resources/public/js/compiled"
                 :asset-path      "/js/compiled"
-                :dev             {:modules          {:app {:entries [devtools.preload{{#10x?}}
-                                                                     day8.re-frame-10x.preload{{/10x?}}{{#re-frisk?}}
-                                                                     re-frisk.preload{{/re-frisk?}}
-                                                                     {{name}}.core]}}{{#10x?}}
-                                  :compiler-options {:closure-defines {re-frame.trace.trace-enabled?        true
+                :modules         {:app {:init-fn {{name}}.core/init
+                                        :preloads [devtools.preload{{#10x?}}
+                                                   day8.re-frame-10x.preload{{/10x?}}{{#re-frisk?}}
+                                                   re-frisk.preload{{/re-frisk?}}]}}{{#10x?}}
+                :dev             {:compiler-options {:closure-defines {re-frame.trace.trace-enabled?        true
                                                                        day8.re-frame.tracing.trace-enabled? true}}{{/10x?}}}
-                :release         {:modules          {:app {:entries [{{name}}.core]}}
-                                  :compiler-options {:optimizations :advanced
-                                                     :pretty-print  false
-                                                     :closure-defines {goog.DEBUG                           false{{#10x?}}
```

Comment:
`goog.DEBUG` is automatically set to `false` for `:release` builds. As is `:pretty-print false` and `:optimizations`.